### PR TITLE
Ivan

### PR DESCRIPTION
**Pull request by AndreyeuIvan**
game-model-n-celery
Created nerdy-command.
Logic is following:
1. Creates request to igb.
2. In a cycle takes the name of a game.
3. Put it into api request of twitter.
4. Saves obj of twitter into DB.
5. in a cycle attributes of the game from igb saved into .json file.
6. Creates fixture.

### CODE REVIEW

File: games/service_igdb.py [hunk 1]
```diff
@@ -130,19 +132,19 @@ def get_game_by_id(self, id: int):
 
 
 IG = IGDBWrapper(_.client_id_twitch, token_twitch)
-
+print(token_twitch)
```
Comment (Satevg) [inline games/service_igdb.py]: remove print please

File: games/views.py [hunk 2]
```diff
@@ -7,33 +6,22 @@
...
 
 
 def detail(request, id):
     """
-    Добавить айди для переменной
+    Предоставляем детали игры
+    """
     try:
-        game = Game.objects.get(pk=game_id)
+        context = {"game": Game.objects.get(pk=id)}
```
Comment (Satevg) [inline games/views.py]: again, better to remove except and use `get_object_or_404`

File: games/management/commands/nerdy.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            new_tweet.save()
+            ids.append(new_tweet.id)
+        return ids
+
+    def handle(self, *args, **options):
+        date = datetime.now().strftime("%Y-%m-%d %H")
+        igb_json = IG.get_games_list()
+        my_format = []
+        for i in igb_json:
+            my_format += [
```
Comment (Satevg) [inline games/management/commands/nerdy.py]: probably more pythonic it'll be like

my_format = [{"model": "games.game", "pk": game.get("id"), "fields": {and so on}} for game in igb_json]

---------------------------------

Optional: enable style-checks/fixes with `black` package.

**Review by Satevg — CHANGES_REQUESTED** [review]

**Review by Satevg — APPROVED** [review]

**Comment by AndreyeuIvan** [comment]
Task to create command-django - done

**Pull request merged**